pkg/storage/utils/metadata/cs3.go

```diff
@@ -420,6 +454,9 @@ func (cs3 *CS3) providerClient() (provider.ProviderAPIClient, error) {
 }
 
 func (cs3 *CS3) getAuthContext(ctx context.Context) (context.Context, error) {
+	ctx, span := appctx.GetTracerProvider(ctx).Tracer(tracerName).Start(ctx, "getAuthContext")
```

Comment:
ah in line 465 we are using context.Background() again. Please double check we are not making the same mistake elsewhere in this PR. I did not check that when initially adding the traces.